tests/1_host_configuration.sh

```diff
@@ -41,7 +41,7 @@ fi
 check_1_6="1.6  - Keep Docker up to date"
 docker_version=$(docker version | grep -i -A1 '^server' | grep -i 'version:' \
   | awk '{print $NF; exit}' | tr -d '[:alpha:]-,')
-docker_current_version="1.7.0"
+docker_current_version="1.8.0"
```

Comment:
Thanks @diogomonica, I totally understand, you've shipped a lot of cool things in the last days.